lexicons/com/atproto/admin/defs.json

```diff
@@ -158,6 +121,59 @@
         }
       }
     },
+    "subjectStatusView": {
+      "type": "object",
+      "required": ["id", "subject", "createdAt", "updatedAt", "reviewState"],
+      "properties": {
+        "id": { "type": "integer" },
+        "subject": {
+          "type": "union",
+          "refs": ["#repoRef", "com.atproto.repo.strongRef"]
+        },
+        "subjectRepoHandle": { "type": "string" },
+        "updatedAt": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Timestamp referencing when the last update was made to the moderation status of the subject"
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Timestamp referencing the first moderation status impacting event was emitted on the subject"
+        },
+        "reviewState": {
+          "type": "ref",
+          "ref": "#subjectReviewState"
+        },
+        "note": {
+          "type": "string",
+          "description": "Sticky note on the subject."
+        },
+        "muteUntil": {
+          "type": "string",
+          "format": "datetime"
+        },
+        "lastReviewedBy": {
+          "type": "string",
+          "format": "did"
+        },
+        "lastReviewedAt": {
+          "type": "string",
+          "format": "datetime"
+        },
+        "lastReportedAt": {
+          "type": "string",
+          "format": "datetime"
+        },
+        "takendown": {
+          "type": "boolean"
+        },
```

Comment:
Might be worth taking a peek at `#statusAttr` and `getSubjectStatus`, `updateSubjectStatus` lexicons— we may be able to make these more consistent with each other.

Reply:
not sure if these can be same though? `statusAttr` is defined as 

```
{
    "applied": { "type": "boolean" },
    "ref": { "type": "string" }
}
```

but in the appview, `takendown: true` always means `applied: true` and `ref` would just be a reference to the entry's id itself so it's kinda redundant unless we map this ref to reference to the event that changed the status?

Reply:
They do also have different names in `getSubjectStatus` / `updateSubjectStatus`: `takedown` vs. `takendown`.